enclave-runtime/src/tls_ra/authentication.rs

```diff
@@ -0,0 +1,118 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+//! Remote attestation certificate authentication of server and client
```

Comment:
I did not change this part, simply copy pasted from the old tls_ra.rs file. So I'm unsure why there are so many unused variables in the function definition. Hence, I left as it is for now.